src/Entity/ProductBundle.php

```diff
@@ -24,6 +24,9 @@ class ProductBundle implements ProductBundleInterface, ResourceInterface, CodeAw
     /** @var Collection */
     private $products;
 
+    /** @var ProductInterface */
+    private $product;
```

Comment:
Not sure if we will later be aware that this product represents the product-entity of the product-bundle-entity. Maybe there is a good chance to be confused with $product and $products...

Reply:
I had the same thoughts but no idea how to name it better. Suggestions?

Reply:
Suggestion: Rename "products" to bundledProducts" and leave "product" as it is

Reply:
bundleProducts and product would be ok for me. 

Btw maybe we should not assign Products directly to the Bundle, somehow we need to know which product we are looking at. E.g. 'do we have a T-Shirt or an Short'. So I thought about doing the relation between bundles and its products through an additional resource, e,g. 'ProductBundleSlots'.

Reply:
After verbal discussion: Current ```$products``` will be removed in future, so this is not relevant by now.